web-components/docs/README.md

```diff
@@ -0,0 +1,412 @@
+# Chart Component APIs
+
+## Including the library
+
+Embed [Data Commons](https://datacommons.org) charts in your web pages including
+datacommons.js and styles in your webpage's `<head></head>` tags. These files
+must be included for the web components to work.
+
+```html
+<link rel="stylesheet" href="https://datacommons.org/css/ranking.min.css" />
+<link rel="stylesheet" href="https://datacommons.org/css/nl_interface.min.css" />
+<script src="https://datacommons.org/datacommons.js"></script>
+```
+
+### Example
+
+```html
+<html>
+  <head>
+    <link rel="stylesheet" href="https://datacommons.org/css/ranking.min.css" />
+    <link rel="stylesheet" href="https://datacommons.org/css/nl_interface.min.css" />
+    <script src="https://datacommons.org/datacommons.js"></script>
+  </head>
+  <body>
+    <!-- Embedded charts will go here -->
```

Comment:
Can you add a real web component example here?

Reply:
Good idea. Added.